src/chainparams.cpp

```diff
@@ -240,7 +239,6 @@ class CTestNetParams : public CChainParams {
         vSeeds.clear();
         // nodes with support for servicebits filtering should be at the top
         vSeeds.push_back(CDNSSeedData("testnet5.b-pay.net", "seed.testnet5.b-pay.net"));
-        vSeeds.push_back(CDNSSeedData("bloqseeds.net", "bitcoin-testnet.bloqseeds.net", false)); // bloq.com seed
```

Comment:
How does your concerns relate to testnet which by definition should never have any value in it's coins?